pep-0705.rst

```diff
@@ -0,0 +1,96 @@
+PEP: 705
+Title: Recording provenance of installed packages
+Author: Fridolin Pokorny <fridolin.pokorny at gmail.com>, Trishank Karthik Kuppusamy <karthik@trishank.com>
+Sponsor:
+PEP-Delegate:
+Discussions-To: https://discuss.python.org/t/pep-705-recording-provenance-of-installed-packages/23340
+Status: Draft
+Type: Process
+Content-Type: text/x-rst
+Created: 30-Jan-2002
+Post-History:
```

Comment:
```suggestion
Post-History: `03-Dec-2021 <https://discuss.python.org/t/pip-installation-reports/12316>`__,
             `30-Jan-2023 <https://discuss.python.org/t/pep-705-recording-provenance-of-installed-packages/23340>`__,
```

You should add the previous thread that spawned this, and the current one you created.